incubator/hnc/Makefile

```diff
@@ -220,7 +220,20 @@ ifndef HNC_REPAIR
 	@sleep 5
 endif
 	go clean -testcache
-	go test -v -timeout 0 ./test/...
+	go test -v -timeout 0 ./test/e2e/...
+
+# This test is for conversion from v1alpha1 to v1alpha2. It will run on any
+# cluster w/o HNC. It will tear down any existing HNC deployment before and
+# after the test.
+#
+# Note the `-timeout 0` that's passed to the `go test` command - by default, a
+# Go test suite has a 10m timeout, and the flag disables that timeout.
+.PHONY: conversion-test
+# Have the manifests and kubectl plugin ready for the current change but do not
+# apply it or copy the plugin binary to the $PATH yet. It will be done in the test.
+conversion-test: docker-push build manifests
```

Comment:
Can we change this to `test-conversion`? Maybe update the existing one to `test-e2e` as well. That way, we can more easily use autocomplete if we forget what test we're trying to run.

Reply:
sg will do.